services/frontend-service/src/ui/SideBar/SideBar.tsx

```diff
@@ -14,23 +14,16 @@ You should have received a copy of the GNU General Public License
 along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
 
 Copyright 2021 freiheit.com*/
-import { useCallback } from 'react';
 import { Button } from '../components/button';
 
-export const SideBar: React.FC = () => {
-    const hideSideBar = useCallback(() => {
-        const sideBarRef = document.getElementsByClassName('mdc-drawer--displayed')[0];
-        if (sideBarRef?.classList.contains('mdc-drawer--displayed')) {
-            sideBarRef?.classList.remove('mdc-drawer--displayed');
-            sideBarRef?.classList.add('mdc-drawer--hidden');
-        }
-    }, []);
+export const SideBar: React.FC<any> = (props: { className: string; reference: any }) => {
```

Comment:
```suggestion
export const SideBar: React.FC<{ className: string; toggleSidebar: () => void }> = (props) => {
```

Reply:
Fixed! I will close the conversation, but will ask why the second option is preferable.